report/sections/Conclusion/conclusion.tex

```diff
@@ -0,0 +1,21 @@
+The main goal of this project was to create an internet-based application that would solve some problem.
+Furthermore, another requirement in the study regulation is that the developed system also needs to be tested for scalability.
+In this project, we developed a system for lecturers of the \textit{Programming Paradigms} course that can provide fast and efficient feedback on student exercise submissions.
+The system allows lecturers to create exercises for students and define accompanying tests for verifying correctness of student solutions.
+
+To create this application, we developed a working user interface that students and lecturers can use in order to interact with the system.
+The user interface was developed with user-friendliness in mind, providing minimal hindrance in accessing and solving exercises.
+A backend was developed in order to process and serve frontend requests.
+Supplementing this, a database was also implemented to store information regarding users and exercises.
+The backend features a component that uses the GHC interpreter to run tests on exercise submissions to provide users with feedback on their submission.
```

Comment:
Please refrain from calling the Test Runner a backend. It is a service that the Next.js backend uses. The reader would be very confused if we use both the Test Runner and Next.js backend as 'backend'.

Reply:
```suggestion
The backend is connected to the Test Runner. The Test Runner uses the GHC interpreter to run tests on exercise submissions to provide users with feedback on their submission.
```